src/rnp/rnpcfg.cpp

```diff
@@ -319,6 +324,10 @@ rnp_cfg_getstr(const rnp_cfg_t *cfg, const char *key)
         return it->val.val._string;
     }
 
+    if (strcmp(key, CFG_HASH) == 0) {
```

Comment:
This seems weird, @ni4 what do you think?

Reply:
@dewyatt Agree, to hacky. I think this check should be moved to place where hash algorithm is checked (or moved to function like `rnp_cfg_gethashalg()`)